Update documentation for the users page (#42)
gatling/frontline-cloud-doc: content/reference/admin/users/index.md

## Managing Users

To access the Users administration, click on the {{< icon user >}} icon in the navigation bar.

{{< img src="users-table.png" alt="Users table" >}}

> To invite a user to your organization, click on the **Create** button and fill the user information:

Reviewer: ```suggestion
To invite a user to your organization, click on the **Create** button and fill in the user information:
```

> - **Github Username**: must match user Github username in order to connect and *cannot be updated*.

Reviewer: ```suggestion
- **Github Username**: must match the user's GitHub username in order to connect and *cannot be updated*.
```

Reviewer 2: ```suggestion
- **GitHub Username**: must match the user's GitHub username in order to connect and *cannot be updated*.
```

> - **Firstname** | **Lastname** | **Email address**: explicit

Reviewer: ```suggestion
- **Firstname** | **Lastname** | **Email address**: explicit. The user will be able to change these once they are connected.
```
(maybe not needed, but since the other fields are either not modifiable at all, or not by the user themselves...)

- **Roles**: either choose a global role, or select none and specify each role by team according to the permissions you want to grant.

> The user invited will have to connect with its Github account. A user can be part of only one organization at the moment.

Reviewer: ```suggestion
The user invited will have to connect with their Github account. A user can join only one organization at the moment.
```

{{< img src="users-create.png" alt="User creation" >}}

> You can edit the user by clicking on the {{< icon pencil-alt >}} icon and remove them from your organization using the checkboxes on the table's right part.

Reviewer: ```suggestion
To edit a user, click on the {{< icon pencil-alt >}} icon. To remove them from your organization, select them using the checkbox on the left of the table and click on the **Remove** button.
```
